src/Codec/Xlsx/Types/StyleSheet.hs

```diff
@@ -771,6 +818,170 @@ idToStdNumberFormat i  = if i < firstUserNumFmtId then Just (NfOtherImplied i) e
 firstUserNumFmtId :: Int
 firstUserNumFmtId = 164
 
+-- | Implied cell styles
+-- Annex G.2, p. 4446
+data ImpliedCellStyle =
+    CStNormal            -- ^> 0 Normal
+  | CStRowLevel          -- ^> 1 RowLevel_ + level #
+  | CStColLevel          -- ^> 2 ColLevel_ + level #
+  | CStComma             -- ^> 3 Comma
+  | CStCurrency          -- ^> 4 Currency
+  | CStPercent           -- ^> 5 Percent
+  | CStComma0            -- ^> 6 Comma [0]
+  | CStCurrency0         -- ^> 7 Currency [0]
+  | CStHyperlink         -- ^> 8 Hyperlink
+  | CStFollowedHyperlink -- ^> 9 Followed Hyperlink 
+  | CStNote              -- ^> 10 Note
+  | CStWarningText       -- ^> 11 Warning Text
+  | CStTitle             -- ^> 15 Title
+  | CStHeading1          -- ^> 16 Heading 1
+  | CStHeading2          -- ^> 17 Heading 2
+  | CStHeading3          -- ^> 18 Heading 3
+  | CStHeading4          -- ^> 19 Heading 4
+  | CStInput             -- ^> 20 Input
+  | CStOutput            -- ^> 21 Output
+  | CStCalculation       -- ^> 22 Calculation
+  | CStCheckCell         -- ^> 23 Check Cell
+  | CStLinkedCell        -- ^> 24 Linked Cell
+  | CStTotal             -- ^> 25 Total
+  | CStGood              -- ^> 26 Good
+  | CStBad               -- ^> 27 Bad
+  | CStNeutral           -- ^> 28 Neutral
+  | CStAccent1           -- ^> 29 Accent1
+  | CSt20Accent1         -- ^> 30 20% - Accent1
+  | CSt40Accent1         -- ^> 31 40% - Accent1
+  | CSt60Accent1         -- ^> 32 60% - Accent1
+  | CStAccent2           -- ^> 33 Accent2
+  | CSt20Accent2         -- ^> 34 20% - Accent2
+  | CSt40Accent2         -- ^> 35 40% - Accent2
+  | CSt60Accent2         -- ^> 36 60% - Accent2
+  | CStAccent3           -- ^> 37 Accent3
+  | CSt20Accent3         -- ^> 38 20% - Accent3
+  | CSt40Accent3         -- ^> 39 40% - Accent3
+  | CSt60Accent3         -- ^> 40 60% - Accent3
+  | CStAccent4           -- ^> 41 Accent4
+  | CSt20Accent4         -- ^> 42 20% - Accent4
+  | CSt40Accent4         -- ^> 43 40% - Accent4
+  | CSt60Accent4         -- ^> 44 60% - Accent4
+  | CStAccent5           -- ^> 45 Accent5
+  | CSt20Accent5         -- ^> 46 20% - Accent5
+  | CSt40Accent5         -- ^> 47 40% - Accent5
+  | CSt60Accent5         -- ^> 48 60% - Accent5
+  | CStAccent6           -- ^> 49 Accent6
+  | CSt20Accent6         -- ^> 50 20% - Accent6
+  | CSt40Accent6         -- ^> 51 40% - Accent6
+  | CSt60Accent6         -- ^> 52 60% - Accent6
+  | CStExplanatoryText   -- ^> 53 Explanatory Text
```

Comment:
What about adding `CStOther Int` to allow additional styles in future versions of the spec?

Reply:
It makes sense.